docs/hashcat-plugin-development-guide.md

```diff
@@ -537,17 +537,21 @@ Additionally there is a couple of command line parameters that you want to use:
 * -d 1: In case you have multi compute devices in your system, limit it to a single compute device. This is to reduce startup and JiT compile time.
 
 Typically a developer command line for hashcat looks the following:
-
 ```
-$ rm -rf kernels $HOME/.nv; ./hashcat -m XXXXX hash.txt word.txt --potfile-disable --self-test-disable -n 1 -u 1 -T 1 --quiet --backend-vector-width 1 -d 1
+$ rm -rf kernels $HOME/.nv; ./hashcat -m XXXXX hash.txt word.txt --potfile-disable --self-test-disable -n 1 -u 1 -T 1 --quiet --backend-vector-width 1 -d 1 --force
 ```
 
 If you need to printf from without a _loop kernel, keep in mind that you need to add a branching manually for a specific loop position.
-
 ```
 if ((loop_pos + i) == 0) printf ("%08x\n", a);
 ```
 
+When using  -n 1 -u 1 -T 1 you shouldn't add
```

Comment:
I agree with @jsteube here: it's better to avoid "shouldn't" and tell the user what NOT to do. Instead, it would make more sense to mention how to do it correclty in the guide and emphasize that the `gid`/`lid` are zero-based.

Please also stick to the coding style we use:

```
if ((gid == 0) && (lid == 0))
{
  printf ("%08x\n", a);
}
```

I would suggest to just show how to do printf and afterwards mention something like this: "Please note that the variables gid and lid are zero-based and therefore we start counting with 0", or something like this.

Thank you very much